packages/card/test.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable no-undef */
+import tap, { test, beforeEach, teardown } from 'tap';
+import { chromium } from 'playwright';
+
+tap.before(async () => {
+  const browser = await chromium.launch({ headless: true });
```

Comment:
Yes, I was able to remove headless and it continues to remain headless. Useful to have it in there in case you want to flip the switch and test things out locally